docs/quickstart.md

```diff
@@ -1,246 +1,67 @@
-
-### Listing leases
-
-To list the leases, use an HTTP GET request to the *${api_url}/leases* endpoint
-to see the response as shown here:
-
-```json
-[
-    {
-        "accountId": "123456789012",
-        "budgetAmount": 20,
-        "budgetCurrency": "USD",
-        "budgetNotificationEmails": [
-            "myuser@example.com"
-        ],
-        "createdOn": 1572381585,
-        "expiresOn": 1572382800,
-        "id": "94503268-426b-4892-9b53-3c73ab38aeff",
-        "lastModifiedOn": 1572381585,
-        "leaseStatus": "Active",
-        "leaseStatusModifiedOn": 1572381585,
-        "leaseStatusReason": "Active",
-        "principalId": "DCEPrincipal"
-    }
-]
-```
-
-### Destroying leases
-
-Leases can automatically expire based on a date or a budget amount, but
-leases may also be administratively destroyed at any time. To destroy
-a lease with the API, send a DELETE request to *${api_url}/leases
-with the following request body:
-
-```json
-{
-    "principalId": "DCEPrincipal",
-    "accountId": "123456789012"
-}
-```
-
-The API response for a successful lease destroy looks like this:
-
-```json
-{
-    "accountId": "519777115644",
-    "budgetAmount": 20,
-    "budgetCurrency": "USD",
-    "budgetNotificationEmails": [
-        "john.doe@example.com"
-    ],
-    "createdOn": 1572381585,
-    "expiresOn": 1572382800,
-    "id": "94503268-426b-4892-9b53-3c73ab38aeff",
-    "lastModifiedOn": 1572442028,
-    "leaseStatus": "Inactive",
-    "leaseStatusModifiedOn": 1572442028,
-    "leaseStatusReason": "Destroyed",
-    "principalId": "jdoe123"
-}
-```
+Deploy DCE and lease an account quickly using the DCE CLI.
+
+1. Download the appropriate executable for your OS from the [latest release](https://github.com/Optum/dce-cli/releases/latest). e.g. for mac, you should download dce_darwin_amd64.zip
+
+1. Unzip and move the executable to a directory on your PATH, e.g.
+
+    ```
+    # Download the zip file
+    wget https://github.com/Optum/dce-cli/releases/download/<VERSION>/dce_darwin_amd64.zip
+
+    # Unzip to a directory on your path
+    unzip dce_darwin_amd64.zip -d /usr/local/bin
+    ```
+
+1. Type `dce init`. Leave all fields blank for now.
+
+1. Deploy DCE using IAM Credentials that have AdministratorAccess
+    ```
+    export AWS_ACCESS_KEY_ID=XXXXXXXXXX
+    export AWS_SECRET_ACCESS_KEY=XXXXXXXXXXXXXXXXXXXX
+    dce system deploy
+    ```
+
+1. Retrieve the DCE API url from API Gateway in your master account, and add it to the dce config file, e.g.
+    ```
+    api:
+        host: abcdefghij.execute-api.us-east-1.amazonaws.com
+        basepath: /api
+    region: us-east-1
+    ```
+
+1. Prepare a second AWS account to be your first "DCE Child Account".
+    - Create an IAM role with `AdministratorAccess` and a trust relationship to your DCE Master Accounts
+    - Create an account alias by clicking the 'customize' link in the IAM dashbodard of the child account. This must not include the terms "prod" or "production".
```

Comment:
>  clicking the 'customize' link in the IAM dashbodard of the child account

I'm not sure I'd know where to find this.
Maybe stick with CLI commands if we can:
https://docs.aws.amazon.com/cli/latest/reference/iam/create-account-alias.html

Reply:
I think the UI directions are good in addition to the link to the docs. Sentence can be rewritten as:

> Create an alias by going to the IAM Dashboard and clicking the "customize" link or by [using the AWS CLI](https://docs.aws.amazon.com/cli/latest/reference/iam/create-account-alias.html).